pkg/bbgo/margin_info_updater.go

```diff
@@ -0,0 +1,91 @@
+package bbgo
+
+import (
+	"context"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/c9s/bbgo/pkg/util/timejitter"
+)
+
+type MaxBorrowableCallbackType func(asset string, amount fixedpoint.Value)
+
+//go:generate callbackgen -type MarginInfoUpdater
+type marginInfoUpdaterType struct {
```

Comment:
should be MarginInfoUpdater?

Reply:
I want it to be private.
And ideally expose it via `NewMarginInfoUpdaterFromSession` or any other constructor if necessary.

Reply:
ok, fixed.